documentation/modules/auxiliary/admin/networking/ubiquiti_config.md

```diff
@@ -59,12 +59,12 @@ resource (unifi_config.rb)> run
 ### db File
 
 ```
-resource (unifi_config.rb)> use auxiliary/admin/ubiquiti/ubiquiti_config
+resource (unifi_config.rb)> use auxiliary/admin/networking/ubiquiti_config
```

Comment:
Same here, why do we have these two prompts here, and then later on we switch over to `msf5 auxiliary(admin/networking/ubiquiti_config)` as a prompt? I don't see anything which would suggest why the `resource (unifi_config.rb)` prompt would be displayed.